app/models/vm/operations/power.rb

```diff
@@ -7,10 +7,27 @@ module Vm::Operations::Power
     api_relay_method :suspend
 
     supports :suspend do
-      msg = unsupported_reason(:control) unless supports_control?
-      msg ||= _('The VM is not powered on') unless vm_powered_on?
+      msg = supports_control_powered_on
```

Comment:
I see that you are trying to DRY things up here, but I'd prefer those support blocks to be selfcontained
This way they are easier to read and change later

So keep this here and duplicate it the same for `:pause`